app/views/system_admin/users/show.html.erb

```diff
@@ -1,69 +1,84 @@
-    <dt class="govuk-summary-list__key">
-      Providers
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <% if @user.providers.any? %>
-        <div class="govuk-!-margin-bottom-3">
-          <% @user.providers.each do |provider| %>
-            <%= govuk_link_to "#{provider.name} - #{provider.code}", provider_path(provider), class: "govuk-!-display-block" %>
-          <% end %>
-        </div>
-      <% end %>
-      <%= govuk_link_to("Add provider", new_user_provider_path(@user), class: "govuk-!-display-block add-provider-to-user") %>
-      </div>
-    </dd>
 </dl>
-  
-<%= govuk_link_to("Delete this user", delete_user_path(@user), class: "link--delete govuk-link--no-visited-state govuk-body") unless @user.system_admin? %>
\ No newline at end of file
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <table id="providers" class="govuk-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Providers</caption>
+      <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Name</th>
+        <th scope="col" class="govuk-table__header">Remove access</th>
+      </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% @user.providers.each do |provider| %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+              <%= govuk_link_to provider.name, provider_path(provider), class: 'provider' %>
+            </td>
+            <td class="govuk-table__cell">
+              <%= govuk_link_to "Remove", edit_user_provider_accessions_path(@user, provider), class: 'remove-access' %>
+            </td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+    <%= render GovukButtonLinkTo::View.new(body: "Add a user to a provider",
+                                           url: new_user_provider_path(@user),
+                                           class_option: "govuk-button--secondary add-user-to-provider") %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class ="govuk-grid-column-full">
+    <table id="lead-schools" class="govuk-table">
+      <caption class="govuk-table__caption govuk-table__caption--m">Lead Schools</caption>
+      <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-table__header">Name</th>
+        <th scope="col" class="govuk-table__header">Remove access</th>
+      </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% @user.lead_schools.each do |lead_school| %>
+          <tr class="govuk-table__row">
+            <td class="govuk-table__cell">
+              <%= govuk_link_to lead_school.name, lead_school_path(lead_school), class: "lead-school" %>
+            </td>
+            <td class="govuk-table__cell">
+              <%= govuk_link_to "Remove", edit_user_lead_school_accessions_path(@user, lead_school), class: 'remove-access' %>
```

Comment:
This link will need hidden text to identify it uniquely when there are multiple remove links. I think we might already have a method for doing hidden text?

Ideally something like:

`Remove<span class="govuk-visually-hidden"> access to {{lead_school.name}}</span>`